DistributionPackages/Neos.NeosIo/Migrations/ContentRepository/Version20250627114423.yaml

```diff
@@ -0,0 +1,10 @@
+comments: 'Flatten badge grid by removing intermittent collection'
+migration:
+  - filters:
+      - type: 'NodeType'
+        settings:
+          nodeType: 'Neos.NeosIo:BadgeGrid'
+    transformations:
+      - type: 'FlattenCollection'
+        settings:
+          collectionNodeName: 'badges'
```

Comment:
After this transformation we still have the old empty collection node or did I miss this? Did you also want to remove it with this PR?